coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -80,10 +93,10 @@ def parse(self):
         """
         if self.language == 'python' and self.docstyle == 'default':
             return self._parse_documentation_with_symbols(
-                (':param ', ':'), (':raises ', ':'), ':return:')
+                (':param ', ':'), (':raises ', ':'), ':return:', PY_DOC_REF)
         elif self.language == 'python' and self.docstyle == 'doxygen':
             return self._parse_documentation_with_symbols(
-                ('@param ', ' '), ('@raises ', ' '), '@return ')
+                ('@param ', ' '), ('@raises ', ' '), '@return ', PY_DOC_REF)
```

Comment:
I expect there will be differences in doxygen.

Reply:
https://www.stack.nl/~dimitri/doxygen/manual/autolink.html says:

All words in the documentation that correspond to a documented class and contain at least one non-lower case character will automatically be replaced by a link to the page containing the documentation of the class. If you want to prevent that a word that corresponds to a documented class is replaced by a link you should put a % in front of the word. To link to an all lower case symbol, use \ref.

Reply:
It would be nearly impossible to parse the references using this mechanism. Also, if we do parse the name, I can't think of a way to infer the reference type (`ref_type`) automatically. 

Should we make this shinx only?